cdap-unit-test/src/test/java/io/cdap/cdap/test/app/AppUsingGetServiceURL.java

```diff
@@ -107,6 +107,32 @@ public void ping(HttpServiceRequest request, HttpServiceResponder responder) thr
       }
     }
 
+    @GET
+    @Path("forward/{namespace}")
+    public void forward(HttpServiceRequest request, HttpServiceResponder responder,
+                        @PathParam("namespace") String namespace) throws IOException {
+      URL serviceURL = getContext().getServiceURL(namespace, AppWithServices.APP_NAME, AppWithServices.SERVICE_NAME);
```

Comment:
As mentioned earlier, I think it's better not to tie these two apps together. More specifically, it would be better to put the 'response' endpoint in this app and updates the app and service names here. That way somebody doesn't have to jump across two apps and the unit test in order to figure out what is happening.

This would also allow you to use the same 2 apps to test system calling default and default calling system.